src/schema/v2/artist/updateArtistMutation.ts

```diff
@@ -14,21 +15,50 @@ import {
 } from "lib/gravityErrorHandler"
 
 interface Input {
+  alternateNames: string[]
+  birthday?: string
   coverArtworkId?: string
+  deathday?: string
   displayName?: string
   first?: string
+  gender?: string
+  hometown?: string
   id: string
   last?: string
+  location?: string
   middle?: string
+  nationality?: string
+}
+
+const inputFields = {
+  alternateNames: { type: new GraphQLNonNull(new GraphQLList(GraphQLString)) },
```

Comment:
With this type in your tests, GraphQL at least expects an empty list, and looks like we're not providing in the tests or am I tripping?

Reply:
Yeah that's right - that's why CI is failing. But what should I do? Should I introduce a breaking change that something is required or remove the non null even though in theory arrays should not be nullable??

Reply:
If you want to satisfy the Gravity input; then leaving that optional with a `defaultValue` of empty array, would suffice.